test/Admin.ts/Tracks.test.js

```diff
@@ -1,16 +1,16 @@
 
```

Comment:
For these tests, can we move them out of the `to-be` folder and into the normal folder (or maybe just in an `admin` folder, not an `admin.ts` folder?)

If we want to temporarily skip them, we can use `describe.skip`?

Reply:
I see now that the names of the folders are referencing Beam. I think that's fine for now but I think we should in the future look to having the API not reference client terminology if that makes sense?